hubble-relay.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM docker.io/library/golang:1.14.2 as builder
+ADD . /go/src/github.com/cilium/cilium
+WORKDIR /go/src/github.com/cilium/cilium/hubble-relay
+RUN make
+
+FROM docker.io/library/alpine:3.11 as certs
+RUN apk --update add ca-certificates
+
+FROM docker.io/library/golang:1.14.2-alpine3.11 as gops
```

Comment:
I think the `update-golang-dockerfiles` rule in the main `Makefile` (which is used to update the Go version in all Dockerfiles) will not match this correctly, i.e. it will strip the `-alpine3.11`.

Reply:
OK, I'll update this section to be aligned with what we have in `contrib/packaging/docker/Dockerfile.runtime` so that the `update-golang-dockerfiles` target will continue to work without any modification.

Reply:
Perfect, thanks.